From de7b5e953e56ecad497878d1113abd8c10d8a453 Mon Sep 17 00:00:00 2001
From: Petr Lautrbach <lautrbach@redhat.com>
Date: Fri, 15 Aug 2025 12:38:04 +0200
Subject: [PATCH] Add /var/lib/fapolicyd to tmpfiles
Content-type: text/plain

On image based system, /var/lib/fapolicyd is not created during rpm
installation but needs to be created during boot using tmpfiles.d

Fixes:
    fapolicyd[1463]: Failed writing db version No such file or directory

Signed-off-by: Petr Lautrbach <lautrbach@redhat.com>
---
 init/fapolicyd-tmpfiles.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/fapolicyd-tmpfiles.conf b/init/fapolicyd-tmpfiles.conf
index 24459d340d3c..84fbaa1cf6ed 100644
--- a/init/fapolicyd-tmpfiles.conf
+++ b/init/fapolicyd-tmpfiles.conf
@@ -1,1 +1,3 @@
 d /run/fapolicyd 0770 root fapolicyd -
+d /var/lib/fapolicyd 770 root fapolicyd -
+Z /etc/fapolicyd - root fapolicyd -
-- 
2.50.1

